app/imports/templates/modals/restore.js

```diff
@@ -26,23 +32,38 @@ Template['modals_restore'].onRendered(function() {
     template.fileError.set(null)
     template.allBids.set(null)
     template.newBids.set(null)
+    template.pendingBids.set(null)
     reader.onload = function(e) {
-      let bids;
+      let bids, data, pendingBids;
       try {
-        bids = JSON.parse(e.target.result); 
+        data = JSON.parse(e.target.result);
+        if (Array.isArray(data)) {
```

Comment:
Shouldn't this call `checkBids`, or have the relevant code extracted into a common method?